### ISSUE TITLE
Cannot enter secret access key when adding new accounts in 0.7.0  

### ISSUE THREAD

**Issue by carljavier** [vibrato/awskeyring#36]
After updating to 0.7.0 , I am getting the following when trying to paste the secret access key. 

secret access key input used to show ****** in previous version.

```
$ awskeyring add new-account
   access key id: AKJSIFFLJLSINEF83LD
   secret access key: Secret Access Key is not 40 chars
   secret access key: Secret Access Key is not 40 chars
   secret access key: Secret Access Key is not 40 chars
```

**Comment by tristanmorgan** [comment]
Hi Carl,
The lack of feedback was a result of changes to the input method and I hoped it wouldn't be too much of a problem. There was a PR upstream [thor #209](https://github.com/erikhuda/thor/pull/209) that was closed in favour of just adding a no-echo option.
I will work at getting a better solution in place soon.
T.

**Comment by tristanmorgan** [comment]
The issue of the broken entry of the secret access key is now fixed in release v0.7.1

keeping open until the text feed back is fixed upstream in [Thor](http://github.com/erikhuda/thor) or patched here.

**Comment by tristanmorgan** [comment]
Calling this one Closed, #41 and #37 address both components.
